Munny_Lamp/Munny_Lamp.ino

```diff
@@ -200,14 +198,7 @@ void accelerometer_check() {
   Serial.print(" \tZ: "); Serial.print(event.acceleration.z);
   Serial.println(" m/s^2 ");
 
-  if (event.acceleration.y < 0) {
-    tilt = 1;
-  }
-  else if (event.acceleration.y >= 0) {
-    tilt = 0;
-  }
-
-  if (tilt==1) {
+  if (event.acceleration.y >= 0) {
```

Comment:
this should be "< 0" to match the existing code? maybe even "< 4" so if your table is wobbly it wont go off :D

Reply:
yes, "<0" is the correct direction based on the build as I made it and the guide photos. Standing upright Y is around 9, totally upside down is -9, so 0 works well for the tipping point. This also allows him to blink if you lay him on his back, or not blink if you lay him on his stomach, just given how big his noggin is :)

Reply:
Sorry, @TheKitty I'm not skilled enough in Github foo to be sure I'll do the right thing trying to clone your fork and push a change to it, if that's even the right way to do it. Could you please change line 201 to:
`if (event.acceleration.y < 0) {`

Reply:
I changed it to event.acceleration.y < 0, we'll see what Travis says